src/Functions/FunctionsConversion.h

```diff
@@ -803,7 +803,7 @@ struct ConvertImpl<DataTypeEnum<FieldType>, DataTypeNumber<FieldType>, Name, Con
     }
 };
 
-static ColumnUInt8::MutablePtr copyNullMap(ColumnPtr col)
+static inline ColumnUInt8::MutablePtr copyNullMap(ColumnPtr col)
```

Comment:
?

Reply:
This emitted a linker warning on Mac and I did not want to open a separate PR for it (see commit message).